### PR TITLE
feat: Covid rules, schedule and participant styling

### LINKED ISSUE #1070: Put corona rules on the website
- FFP2 masks are required
- 80 people max
- Everybody needs to be tested (24h before start of entry)

### LINKED ISSUE #1130: Add schedule to home screen

### LINKED ISSUE #1166: Add Friday/Saturday count to the participants page

### PR DESCRIPTION
**Pull request by Narigo**
- Resolves #1070
- Resolves #1130
- Resolves #1166

A bit of main page style changes as well. Hope it's still readable and looks a bit nicer

### CODE REVIEW

**Comment by wolframkriesing** [comment]
looking really good. I leave merging to you, since I merged a number of participants and will leave potential merge conflicts to you ... feel free to merge

**Review by wolframkriesing — APPROVED** [review]

**Comment by Narigo** [comment]
> looking really good. I leave merging to you, since I merged a number of participants and will leave potential merge conflicts to you ... feel free to merge

Thanks, I appreciate the work 😜 Let's wait for a quick review from @lowsky - if he's okay with the rebased version (all participants you recently merged should show up there), he can merge it directly

**Comment by lowsky** [comment]
I love it

**Pull request merged**
